modules/Bio/EnsEMBL/Production/Pipeline/Xrefs/Base.pm

```diff
@@ -256,6 +265,25 @@ sub get_taxon_id {
   return $species_id;
 }
 
+sub get_division {
```

Comment:
I think this would be better named as get_division_id, because that's what it's returning; as it is, it's easy to confuse with the meta_container->get_division method (which this method itself uses...)